OmniKit/Model/AlertSlot.swift

```diff
@@ -99,13 +99,24 @@ public enum PodAlert: CustomStringConvertible, RawRepresentable, Equatable {
         case .finishSetupReminder:
             return AlertConfiguration(alertType: .slot7, duration: .minutes(55), trigger: .timeUntilAlert(.minutes(5)), beepRepeat: .every5Minutes, beepType: .bipBeepBipBeepBipBeepBipBeep)
         case .expirationAlert(let alertTime):
-            return AlertConfiguration(alertType: .slot3, duration: 0, trigger: .timeUntilAlert(alertTime), beepRepeat: .every1MinuteFor3MinutesAndRepeatEvery15Minutes, beepType: .bipBeepBipBeepBipBeepBipBeep)
+            if alertTime == 0 {
```

Comment:
```
let active = alertTime != 0 // disable if alertTime is 0
return AlertConfiguration(alertType: .slot3, active: active, duration: 0, trigger: .timeUntilAlert(alertTime), beepRepeat: .every1MinuteFor3MinutesAndRepeatEvery15Minutes, beepType: .bipBeepBipBeepBipBeepBipBeep)
```
This makes it clear how the check is being used and saves the reader looking through all the other arguments to see if there is anything else different.

Reply:
Done